frontend/components/events/RecommendedEvents.tsx

```diff
@@ -8,20 +8,49 @@ import ChevronLeftButton from "../utility/ChevronLeftButton";
 import ChevronRightButton from "../utility/ChevronRightButton";
 import EventCard from "./EventCard";
 
+const NO_RECOMMENDED_EVENTS = 6;
+
 interface RecommendedEventsProps {
   eventData?: EventData;
 }
 
+const sortByProximityFromHere = (a: EventData, b: EventData, here: EventData | undefined) => {
```

Comment:
more descriptive variables please, and code comments, this function is very logic/ maths heavy.

Please annotate it for future reference. thank you sir